tools/tenscan/backend/webserver/webserver_routes_items.go

```diff
@@ -17,6 +17,13 @@ func routeItems(r *gin.Engine, server *WebServer) {
 	r.GET("/items/batches/", server.getBatchListing)
 	r.GET("/items/blocks/", server.getBlockListing)
 	r.GET("/info/obscuro/", server.getConfig)
+	r.POST("/info/health/", server.getHealthStatus)
+}
+
+func (w *WebServer) getHealthStatus(c *gin.Context) {
+	healthStatus, err := w.backend.GetHealthStatus()
+
+	c.JSON(http.StatusOK, gin.H{"result": healthStatus, "errors": fmt.Sprintf("%s", err)})
```

Comment:
This might have connection errors from the obsclient, but I'm not sure we can easily tell connection errors from other errors here. Worth making a comment tho.